src/index.js

```diff
@@ -0,0 +1,94 @@
+import getToDos, { addTodo, deleteToDo, updateTodo } from './crud.js';
+import checkoffItem from './interactiveList.js';
+import './style.css';
+
+const todoContainer = document.querySelector('.todo-list');
+const clearListBtn = document.querySelector('.btn-clear');
+
+const itemMarkupGen = (data) => `<li class="todo" data-index="${data.index}" data-completed="${data.completed}"> 
+  <label for="${data.index}"> <input type="checkbox" class="checkbox"></label>
+  <input type="text" id="${data.index}" class="item-description-input" name="${data.index}" value="${data.description}">
+   <div class="fa-list-icon"><i
+  class="fa-solid fa-ellipsis-vertical"></i></div> 
+  </li>`;
+
+const populateList = (arr) => {
+  let listString = '';
+
+  const sortedArr = arr.sort((a, b) => a.index - b.index);
+
+  sortedArr.forEach((item) => {
+    listString += itemMarkupGen(item);
+  });
+
+  todoContainer.innerHTML = listString;
```

Comment:
[Optional] Use the` insertAdjacentHTML` method for better performance as this method will append the item to the list while `innerHTML` method removes the entire list and then adds a new list with the newly added item.
You can read more about [insertAdjacentHTML vs innerHTML].